Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/AppCmdOnTargetMachines.ps1

```diff
@@ -208,13 +209,13 @@ function Add-SslCert
     }
     else
     {
-        $showCertCmd = [string]::Format("netsh http show sslcert ipport=0.0.0.0:{0}", $port)
+        $showCertCmd = [string]::Format("netsh http show sslcert ipport={0}:{1}", $ipAddress, $port)
```

Comment:
addressed in commit [165b25c](https://github.com/mmikkelson/vsts-rm-extensions/commit/165b25c5820b95f0dc803b0b1a948d397a9a8a94)